lib/api/objectPutPart.js

```diff
@@ -336,6 +337,7 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                 error: err,
                 method: 'objectPutPart',
             });
+            monitoring.promMetrics('PUT', bucketName, err.code);
```

Comment:
'objectPutPart'?

Reply:
Following naming convention of pushMetrics

Reply:
added